package.json

```diff
@@ -31,6 +31,7 @@
     "babel-preset-es2015": "^6.6.0",
     "chai": "^3.4.0",
     "eslint": "^2.9.0",
+    "enhanced-resolve": "^3.4.1",
```

Comment:
Is this needed as direct `devDependency` when `webpack` is already 'installed' ?

Reply:
@michael-ciniawsky for tests locally

Reply:
Sure, but `enhanced-resolve` is installed via `webpack` already right and flatten in `node_modules` ? Not important... :). Could you please rebase/generate a new lockfile when time ?